OpenRA.Mods.Common/Traits/DetectCloaked.cs

```diff
@@ -24,9 +26,29 @@ public class DetectCloakedInfo : ConditionalTraitInfo
 		public override object Create(ActorInitializer init) { return new DetectCloaked(this); }
 	}
 
-	public class DetectCloaked : ConditionalTrait<DetectCloakedInfo>
+	public class DetectCloaked : ConditionalTrait<DetectCloakedInfo>, INotifyCreated
 	{
+		IDetectCloakedModifier[] rangeModifiers;
+
 		public DetectCloaked(DetectCloakedInfo info)
 			: base(info) { }
+
+		void INotifyCreated.Created(Actor self)
```

Comment:
Doesn't this explicitly override `ConditionalTrait.INotifyCreated.Created` and should have been a `protected override Created` method starting with `base.Created` with no `INotifyCreated`  interface implementation within this file?

Reply:
Good catch.
Should have been

```
		protected override void Created(Actor self)
		{
			rangeModifiers = self.TraitsImplementing<IDetectCloakedModifier>().ToArray();

			base.Created(self);
		}
```

Reply:
This is also a problem for `AutoTarget`, `PrimaryBuilding`, `GrantConditionOnDeploy`, `Explodes`, `KillsSelf`, `ReloadAmmoPool`, `ActorSpawnManager`, `Chronoshiftable`, `ResourcePurifier`.

I am working on a lint check that will prevent this from happening again in the future.